include/vapor/ControlExecutive.h

```diff
@@ -123,13 +123,34 @@ class RENDER_API ControlExec : public MyBase {
  //
  int NewVisualizer(string name);
 
+ //! Return tag pushed to the undo stack when NewVisualizer() is called
+ //
+ string GetNewVisualizerUndoTag() const {
```

Comment:
Just a question, not trying to nitpick:  Why does the ControlExecutive handle tags as hard coded strings, as opposed to the static const strings as is done with the Params classes?

Reply:
They are different "tags". The params classes required strings that were visible at compile time (at least it did when they were first implemented). There is no such requirement in this instance, so a traditional,  "safer" getter was used.